sdk-extensions/aws/src/main/java/io/opentelemetry/sdk/extension/aws/resource/JdkHttpClient.java

```diff
@@ -134,4 +134,16 @@ private static KeyStore getKeystoreForTrustedCert(String certPath) {
       return null;
     }
   }
+
+  private static void readTo(@Nullable InputStream is, ByteArrayOutputStream os)
```

Comment:
A bit of a shame that we have to duplicate this. But I guess that is the best way, we don't want to be in the business of providing some public utils artifact.

Reply:
I'll migrate `Ec2Resource` to use this class instead of duplicating logic soon :)